merchants/verkkokauppa/helpers.py

```diff
@@ -69,8 +69,8 @@ def get_meta_label(key: str, reservation: Reservation) -> str:
 def get_verkkokauppa_order_params(reservation: Reservation) -> CreateOrderParams:
     reservation_unit = reservation.reservation_unit.first()
     quantity = 1  # Currently, we don't support quantities larger than 1
-    price_net = round_decimal(Decimal(quantity * reservation.price_net), 2)
-    price_vat = round_decimal(Decimal(quantity * reservation.price_net * (reservation.tax_percentage_value / 100)), 2)
+    price_net = round_decimal(quantity * reservation.price_net, 2)
```

Comment:
The formula should be:
1. gross price = quantity * reservation.price # e.g. 1 * 123.45 €
2. net price = round_decimal(gross price / (1 + reservation.tax_percentage_value / 100), 2) # e.g. 123.45 / 1.255 = 98.366... ~= 98.37 € -- or in practice reservation.price_net as long as quantity is always 1
3. vat amount = gross - net price # e.g. 123.45 - 98.37 = 25.08 €